src/steps/build-account-to-member-mapping/index.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+  Entity,
+  Relationship,
+  createIntegrationRelationship,
+  JobState,
+} from '@jupiterone/integration-sdk';
+import {
+  STEP_ID as MEMBER_STEP,
+  ACCOUNT_MEMBER_TYPE,
+} from '../fetch-enterprise-account-members';
+import {
+  STEP_ID as ACCOUNT_STEP,
+  ACCOUNT_TYPE,
+} from '../fetch-enterprise-accounts';
+
+const step: IntegrationStep = {
+  id: 'build-account-to-member-relationships',
+  name: 'Build Account-to-Member Relationships',
+  types: [],
+  dependsOn: [ACCOUNT_STEP, MEMBER_STEP],
+  async executionHandler({ jobState }: IntegrationStepExecutionContext) {
+    const accountIdMap = await createAccountIdMap(jobState);
+
+    await jobState.iterateEntities(
+      { _type: ACCOUNT_MEMBER_TYPE },
+      async (member) => {
+        const account = accountIdMap.get(member.enterpriseAccountId as string);
+
+        if (account) {
+          await jobState.addRelationships([
+            createAccountToMemberRelationship(account, member),
+          ]);
+        }
+      },
+    );
+  },
+};
+
+export default step;
+
+async function createAccountIdMap(
+  jobState: JobState,
+): Promise<Map<string, Entity>> {
+  const accountIdMap = new Map<string, Entity>();
```

Comment:
Is there a reason to use Map over a simple JavaScript object?

Reply:
I just think `Map` is a stronger type definition than an untyped `Object`. TBH I borrowed this looking at Charlie's `graph-trend-micro` project, which obviously had a stronger need for a `Map` because the `id` key was actually a `number`, not a `string` (like it is here).

Reply:
I don't think you have to change it but objects can be typed. `Promise<{ [id: string]: Entity }>`

Reply:
Oh, nice. Did not know that!